### ISSUE TITLE
A "change server" button?

### ISSUE THREAD

**Issue by feeltheburn** [raduprv/Eternal-Lands#38]
(Esp. now with the Android not being as simple to switch.)

I was thinking something along the lines of a button on the login screen to "change server". 

Clicked it gives some form of list of the names given to all server options in servers.lst. Select one, and the client disconnects from the current server, connecting to the other one.

**Comment by pjbroad** [comment]
The difficulty has always been that by the time you have the login screen, many files and settings have already been loaded from the starting configuration directory.  Either we do a significant rewrite of the start-up code and clean it up a lot, or we do a full client restart.

**Comment by feeltheburn** [comment]
Ah. Forgot about the different settings, despite that I have different server setups specifically for that. (Doh!)

Was honestly just thinking about the Android version, with it not having window resize issues and such that are needed pre-login screen.

I'm looking for a simpler way to get on the Test server on the Android client (for simpler testing), but at the same time this could be relevant to making sure PK server is accessible as well on it, when it goes live.


And this may end being something only for the Android, then. I don't think separate settings will be needed on it, which could make it possible there at least.

On comp, just setting up some desktop buttons with the proper commands works, but if there's a way to do that on Android, I have no idea.

**Comment by raduprv** [comment]
You can modify the servers list, and put the main server to point to the 
test server.

**Comment by feeltheburn** [comment]
Yeah, done a lot of that on the comp, so figured it would work on Android as well. 

Thinking of something more for the final release though, which in turn would make it simpler for this testing as well. I know the PK server's mostly dead but those still using it may wanna have an Android option as well. (Plus who knows, maybe it'll get some more players, too, hehe.)

Release version could not have test server in the server list, those of us who need it can hand-add it. Just Main and PK options.

**Comment by raduprv** [comment]
That is something to consider on Android, but it would be the very last 
thing I would do, after I finish all the other stuff, including some 
optimizations, etc. I can always release two APKs anyway, one for the PK 
server.

**Comment by feeltheburn** [comment]
True. Just a habit of thinking too far ahead here. ;-)

**Comment by pjbroad** [comment]
Doing this at runtime is still problematic.  Is the addition of the default server option enough? 402097b4213fb0f036a392032695b500d30dc79d
